Directory.Build.props

```diff
@@ -17,7 +16,12 @@
     <IncludeSource>True</IncludeSource>
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
-    <PackageRequireLicenseAcceptance>False</PackageRequireLicenseAcceptance>
+    <PackageProjectUrl>https://tasty.xenial.io</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/xenial-io/Tasty</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <PackageLicenseFile>LICENSE</PackageLicenseFile>
```

Comment:
LICENSE file should have .md extension.
https://docs.microsoft.com/en-us/nuget/reference/nuspec#license

Reply:
They can but not have to. Just checked, license acceptance works, and it will be linked as text which is fine I guess.

Reply:
```xml
    <requireLicenseAcceptance>true</requireLicenseAcceptance>
    <license type="file">LICENSE</license>
```

Reply:
My editor would thank you for letting him know it is markdown content 😉

Reply:
Okay good point. (I use VSCode for most of the metadata stuff, so I didn't mind so much) should be fixed